blocks/card-carousel/card-carousel.js

```diff
@@ -0,0 +1,172 @@
+      div.append(a);
+      a.innerHTML = row.innerHTML;
+
+      careerSlides.append(div);
+      slideDivs.push(div);
+    });
+  } else {
+    for (let i = 0; i < data.length; i += 1) {
+      const div = document.createElement('div');
+      div.classList.add('career-card');
+      const a = document.createElement('a');
+      a.href = data[i].path;
+      const pic = createOptimizedPicture(data[i].image, data[i].pagename);
+      a.append(pic);
+
+      const bqc = document.createElement('div');
+      bqc.classList.add('career-card-bqc');
+      const bq = document.createElement('blockquote');
+      bq.textContent = data[i]['career-quote'];
+      bqc.append(bq);
+      a.append(bqc);
+
+      const nm = document.createElement('h6');
+      nm.textContent = data[i].pagename;
+      a.append(nm);
+
+      const role = document.createElement('p');
+      role.textContent = data[i]['career-jobtitle'];
+      a.append(role);
+
+      const link = document.createElement('button');
+      link.textContent = placeholders['career-carousel-readmore'];
+      const arrow = document.createElement('img');
+      arrow.src = '/icons/angle-right-blue.svg';
+      arrow.alt = 'Go to testimonial';
+      arrow.classList.add('icon-angle-right-blue');
+      link.append(arrow);
+      a.append(link);
+      div.append(a);
+
+      careerSlides.append(div);
+      slideDivs.push(div);
+    }
+  }
+  careerSlider.append(careerSlides);
+
+  const navBar = document.createElement('div');
+  navBar.classList.add('career-slides-navbar');
+  const navButtons = document.createElement('div');
+  navButtons.classList.add('career-slides-nav');
+
+  const buttons = [];
+  for (let i = 0; i < data.length && i < MAX_BUTTONS; i += 1) {
+    const prevDiv = i > 0 ? slideDivs[i - 1] : null;
+
+    const s = document.createElement('span');
+    s.classList.toggle('active-nav', i === 0);
+    s.tabIndex = '-1';
+    s.onclick = () => scrollToCard(i, slideDivs[i], prevDiv, careerSlides, s, document);
+    navButtons.append(s);
+
+    buttons.push(s);
+  }
+  const la = document.createElement('img');
+  la.src = '/icons/angle-left-blue.svg';
+  la.alt = 'Previous person card';
+  la.classList.add('btn-angle');
+  la.onclick = () => scrollToAdjacent(buttons, slideDivs, careerSlides, false, document);
+  navButtons.prepend(la);
+
+  const ra = document.createElement('img');
+  ra.src = '/icons/angle-right-blue.svg';
+  ra.alt = 'Next person card';
+  ra.classList.add('btn-angle');
+  ra.onclick = () => scrollToAdjacent(buttons, slideDivs, careerSlides, true, document);
+  navButtons.append(ra);
+  navBar.append(navButtons);
+
+  block.append(careerSlider);
+  block.append(navBar);
+
+  if (block.children) {
+    // Remove all elements preceding the career slider div
+    while (careerSlider.previousElementSibling) {
+      careerSlider.parentNode.removeChild(careerSlider.previousElementSibling);
+    }
+  }
+
+  document.onkeydown = (e) => {
+    switch (e.keyCode) {
```

Comment:
Seems this is deprecated
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
Could we instead use this?
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code